src/coreComponents/physicsSolvers/wavePropagation/shared/WaveSolverBase.cpp

```diff
@@ -335,8 +337,9 @@ void WaveSolverBase::postInputInitialization()
 
   if( m_useDAS != WaveSolverUtils::DASType::none )
   {
-    GEOS_LOG_LEVEL_RANK_0( 1, "Modeling linear DAS data is activated" );
-    GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "Linear DAS formulation: {}", m_useDAS == WaveSolverUtils::DASType::strainIntegration ? "strain integration" : "displacement difference" ) );
+    GEOS_LOG_LEVEL_RANK_0( logInfo::DASType, "Modeling linear DAS data is activated" );
```

Comment:
How is this working since you seem to have deleted this Macro?

Reply:
nevermind, I found the new one. I don't know why the search is not working properly.